data/human/human missions.txt

```diff
@@ -6849,6 +6849,133 @@ mission "FW Refugees to Humanika"
+			choice
+				`	"So you search for abandoned ships and loot them?"`
+				`	"I get it, you're recycling old ships for profit."`
+			label precisely
+			`	"Precisely. If you want to go hunting for derelicts yourself, they can be rarely found in out of the way uninhabited systems. Good hunting out there." He turns back to his score.`
+				decline
+
+mission "Derelict Treasure Map"
+	description "Follow the treasure map you found in a derelict ship."
+	landing
+	to offer
+		has "derelict treasure map"
+	destination "New Argentina"
+	on offer
+		conversation
+			`Upon landing on <origin> you take out the treasure map you recovered from the derelict ship. The map gives detailed instructions about where the treasure was buried on the sparsely populated world of <destination>. However, the map's creator claims to have obtained some kind of alien weapons that disrupts a ship's electrical systems. The outlandish claim seems to bring doubt to the validity of the map.`
+			choice
+				`	(Follow the map.)`
+					accept
+				`	(Throw the map away, it's probably fake.)`
+					decline
+	on complete
+		conversation
+			`Flying low over <planet> you follow the instructions to an isolated valley where you promptly land and get to digging. After hours of shovelling the hard soil you're met with a thunk as you strike metal.`
+			branch hai
+				has "First Contact: Hai: offered"
+			`	Opening the cache reveals a large store of credits along with the alien weapons the map's owner wrote about. You marvel at their strange design and wonder at their power.`
+			choice
+				`	(Loot the treasure.)`
+					goto end
+			label hai
+			`	Opening the cache reveals a large store of credits along with two Hai Ion Cannons. These must be the alien weapons the map's owner wrote about.`
+			choice
+				`	(Loot the treasure.)`
+					goto end
+			label end
+			action
+				payment 386000
+				outfit "Ion Cannon" 2
+			`	You grab the <payment> credits, and with the assistance of some machinery you bring the weapons back to your ship.`
+
+mission "Derelict Family Investment 1"
+	name "Lost Family Investment"
+	description "Return the credits you found in a derelict ship to the family it belongs to."
+	name "Colin Sawyer's Investment"
+	landing
+	to offer
+		has "derelict family investment"
+	destination "Sundive"
+	on offer
+		conversation
+			`Upon landing on <origin> you take out the text log you recovered alongside the case of credits from the derelict ship. The log reveals the credits were the result of a risky investment in an attempt to buy a better life for the captain of the ship, Colin Sawyer, and his family. You assume that the captain never made it home, and looking further into the personal log you find that the family used to reside on <destination>.`
```

Comment:
Again, say how many credits it is. We can't really make a decision if we don't know what we're dealing with.